File: 2026-08/src/day_22/prms_181942/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

string solution(string str1, string str2) {
    string s;
    int idx = 0;

    while (idx < str1.size()) {
        s += str1[idx];
        s += str2[idx];
        idx++;
    }

    return s;
}

File: 2026-08/src/day_22/prms_181942/Solution.java
package day_22.prms_181942;

class Solution {
    public String solution(String str1, String str2) {
        StringBuilder sb = new StringBuilder();
        int idx = 0;

        while (idx < str1.length()) {
            sb.append(str1.charAt(idx)).append(str2.charAt(idx));
            idx++;
        }

        return sb.toString();
    }
}
